Classes/Controller/Backend/Order/OrderController.php

```diff
@@ -124,6 +130,8 @@ public function exportAction(): ResponseInterface
     public function showAction(Item $orderItem): ResponseInterface
     {
         $this->moduleTemplate = $this->moduleTemplateFactory->create($this->request);
+        $buttons = $this->getOrderButtons($orderItem);
```

Comment:
We had this some lines above. `$buttons` is only used one time, so we can remove this extra variable. I'll merge this and fix it later.